tasks/winbuild.py

```diff
@@ -9,12 +9,17 @@
 from tasks.msi import build_installer as build_installer_msi
 from tasks.omnibus import build as omnibus_build
 
-# Output directory for package files
-OUTPUT_PATH = os.path.join(os.getcwd(), "omnibus", "pkg")
 # Omnibus stores files here, e.g. C:\opt\datadog-agent, C:\opt\dataog-installer
 OPT_SOURCE_DIR = os.path.join('C:\\', 'opt')
 
 
+def _get_output_path():
+    output_path = os.path.join(os.getcwd(), "omnibus", "pkg")
+    if 'CI_PIPELINE_ID' in os.environ:
+        output_path = os.path.join(output_path, f'pipeline-{os.getenv("CI_PIPELINE_ID")}')
```

Comment:
```suggestion
        output_path = os.path.join(output_path, f'pipeline-{os.environ["CI_PIPELINE_ID"]}')
```
I prefer to keep this syntax everywhere even though the variable is checked